build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+TOOLCHAIN=rm-toolchain
+source $TOOLCHAIN/environment-setup-cortexa9hf-neon-oe-linux-gnueabi && \
```

Comment:
Any reason you are using `&&` instead of just running `set -e` at the start like you do in `draft-rm-install` and `draft-rm-uninstall`?

Reply:
Good point, this is messy. Fixed and added `set -x` too.